chaco/plot.py

```diff
@@ -173,8 +173,8 @@ def __init__(self, data=None, **kwtraits):
             elif type(data) in (ndarray, tuple, list):
                 self.data = ArrayPlotData(data)
             else:
-                raise ValueError, "Don't know how to create PlotData for data" \
-                                  "of type " + str(type(data))
+                raise ValueError("Don't know how to create PlotData for data"
+                                 "of type {}".format(type(data)))
```

Comment:
Do we care about Python 2.6? I assume not, but we should probably make that explicit somewhere.

(If we do care about Python 2.6, this needs to be `{0}` rather than `{}`.)